File: StdCore.Web/StdCore.Web.LoadersScheduler/Data/Enum/TaskType.cs
using System.ComponentModel;

namespace Std.WebClient.Data.Enum
{
    public enum TaskType
    {
        [Description("План")]
        Plan = 1,
        [Description("ФМ")]
        Unexpected = 2,
        [Description("Перерыв")]
        Break = 3
    }
}

File: StdCore.Web/StdCore.Web.LoadersScheduler/ClientApp/src/data/enum/task-type.ts
export enum TaskType {
    Plan = 1,
    Unexpected = 2,
    Break = 3
}

export const TaskTypeName = {
    [TaskType.Plan]: "Плановая задача",
    [TaskType.Unexpected]: "ФМ",
    [TaskType.Break]: "Перерыв"
};
